e2e-tests/functions

```diff
@@ -603,7 +612,7 @@ wait_for_delete() {
 		echo -n .
 		let retry+=1
 		if [ $retry -ge $wait_time ]; then
-			kubectl_bin logs ${OPERATOR_NS:+-n $OPERATOR_NS} $(get_operator_pod) \
+			kubectl logs ${OPERATOR_NS:+-n $OPERATOR_NS} $(get_operator_pod) \
```

Comment:
Why do we change kubectl_bin to kubectl here?

Reply:
In some cases I have found that `kubectl_bin` can mask the original issue, but I don't remember what was the exact reason to change it here, I can try to change back and we can see.